packages/admin-ui/src/filters/filter-status.tsx

```diff
@@ -22,8 +23,38 @@ export function FilterStatus(props: FilterStatusProps) {
 
   return (
     <>
+      {status === 'empty' && (
+        <Box csx={style.statusLayout}>
+          <Text variant="title2">{formatMessage('empty')}</Text>
+        </Box>
+      )}
+
+      {status === 'loading' && (
+        <Stack csx={style.statusLayout} space="$2xl">
+          <Stack space="$xl" csx={{ width: '3/4' }}>
+            <Skeleton csx={{ height: 20 }} />
+            <Skeleton csx={{ width: '9/12', height: 20 }} />
+            <Skeleton csx={{ width: '10/12', height: 20 }} />
+            <Skeleton csx={{ width: '11/12', height: 20 }} />
+            <Skeleton csx={{ width: '8/12', height: 20 }} />
+          </Stack>
+
+          <Stack
+            direction="row"
```

Comment:
Shouldn't it be an `Inline` then?

Reply:
Inline didn't work with the child selector